internal/kafka/internal/workers/kafka_mgrs/preparing_kafkas_mgr.go

```diff
@@ -60,6 +60,7 @@ func (k *PreparingKafkaManager) Reconcile() []error {
 	for _, kafka := range preparingKafkas {
 		glog.V(10).Infof("preparing kafka id = %s", kafka.ID)
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusPreparing, kafka.ID, kafka.ClusterID, time.Since(kafka.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusPreparing, kafka.ID, kafka.ClusterID)
```

Comment:
I think if a reconciliation occurs for a given kafka X, the metric will be updated:
I am wondering if a better strategy might be to set this status when the Kafka transition state: e.g when it moves from `accepted` to `preparing`: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/ee6c732df5c8a67e06a5c2c74e26705e85d2c253/internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go#L121 we fire this metric: 		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusPreparing, kafka.ID, kafka.ClusterID)